README.md

```diff
@@ -131,7 +131,7 @@ $ curl -s -X POST "localhost:3999/extended/v1/faucets/stx?address=$(cat ./cli_ke
 3. Check your balance:
 
 ```bash
-$ curl -s "localhost:3999/v2/accounts/$(cat ./cli_keychain.json | jq -r .keyInfo.address)?proof=0" | jq -r .balance
+$ curl -s "https://stacks-node-api.regtest.stacks.co/v2/accounts/$(cat ./cli_keychain.json | jq -r .keyInfo.address)?proof=0" | jq -r .balance
```

Comment:
`localhost` here is required to be used since this network doesn't sync with the external testnet. i.e. requesting stx from the external faucet won't be available on a private-testnet.